src/analysis/timeline/timelineCred.test.js

```diff
@@ -57,31 +58,36 @@ describe("src/analysis/timeline/timelineCred", () => {
     const tc = exampleTimelineCred();
     const json = exampleTimelineCred().toJSON();
     const tc_ = TimelineCred.fromJSON(json, credConfig());
+    const tcv = getDefaultTimelineCredView(tc);
+    const tcv_ = getDefaultTimelineCredView(tc_);
     expect(tc.graph()).toEqual(tc_.graph());
     expect(tc.params()).toEqual(tc_.params());
     expect(tc.config()).toEqual(tc_.config());
-    expect(tc.credSortedNodes(NodeAddress.empty)).toEqual(
-      tc.credSortedNodes(NodeAddress.empty)
+    expect(tcv.credSortedNodesView(NodeAddress.empty)).toEqual(
+      tcv_.credSortedNodesView(NodeAddress.empty)
     );
   });
 
   it("cred sorting works", () => {
```

Comment:
These tests no longer apply to the `timelineCred` class, and should be moved to `timelineCredView.test.js`